### DISCUSSION TITLE
RTKPOST PPP Static Only Work for GPS

### DISCUSSION THREAD

**Discussion by AndDufresne** [tomojitakasu/RTKLIB#734] [Q&A]
I'm new to this field. I want to evaluate the differences of GNSS constellation combinations of GPS, GLONASS, Galileo, and BeiDou. There are 15 sets of these 4 constellation combinations (GPS-only, GPS & GLONASS, GLONASS & Galileo & BeiDou, ...).
I downloaded GNSS observation and navigation data from CDDIS.
I'd like to ask...
1. Is PPP-static of RTKPOST ver.2.4.3 b34 only work for GPS? There is no output result for other constellations. Or is there a specific setting for another 3 constellations? My setting now is as the pic below.
2. However, if I set the ephemeris/clock to broadcast, the program can execute all combinations. Is the result of PPP-static using broadcast ephemeris/clock valid? Can it still be called PPP when not using precise ephemeris/clock?
![image](https://github.com/tomojitakasu/RTKLIB/assets/87031718/c272e9a9-fc8c-4aa0-b68a-f1b295c6b645)
![image](https://github.com/tomojitakasu/RTKLIB/assets/87031718/48214bef-6597-4331-99dc-852a62d49e9b)
![image](https://github.com/tomojitakasu/RTKLIB/assets/87031718/6647e629-6ad0-466d-8e1f-1232577d8ae6)

**Comment by sharkbig** [comment]
I have the same problem. Do you solve this problem?

**Reply by AndDufresne** [reply]
Nope, I'm using another open-source program instead...
PPPH is also not working for me.
You can try using MG-APP.

**Reply by sharkbig** [reply]
Thanks for suggestion. And I solved the problem with just changing the suffix of clk and sp3 file from capital to small letter.
